src/e3/testsuite/driver/classic.py

```diff
@@ -317,7 +324,31 @@ def tear_down(self) -> None:
         See set_up's docstring for the rationale behind this API.
         """
         if self.working_dir_cleanup_enabled:
-            rm(self.working_dir(), True)
+            # TODO (U222-013): sometimes on Windows, the recursive rm fails. In
+            # such cases, create a dedicated error message and dump the content
+            # of the working directory tree to help investigation.
```

Comment:
Should we tell @kokkonisd that we don't need this feature in core e3 then?

Reply:
Dimitri is in copy of my last mail on the ticket (where I announce that I’m doing this), and we haven’t heard from him since the ticket was assigned to him, so I think it’s fine. In any case, I see no harm coming from parallel work on e3-core. Thanks for the review!

Reply:
Works for me - thank you for implementing this! :)